frontend/src/@types/bot.d.ts

```diff
@@ -62,7 +76,9 @@ export type RegisterBotRequest = {
   title: string;
   instruction: string;
   description?: string;
-  embeddingParams?: EmdeddingPrams;
+  embeddingParams?: EmdeddingParams;
```

Comment:
Please keep the consistency. In this case we have nullable api endpoint, so it would be:

```ts
  embeddingParams?: EmdeddingParams;
  generationParams?: GenerationParams;
  searchParams?: SearchParams;
```